repl/evaluator_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+)
+
+func TestEvalSimple(t *testing.T) {
+	eval, err := NewEvaluator()
+	if err != nil {
+		panic(err)
```

Comment:
Prefer `t.Fatalf("NewEvaluator() failed: %v", err)` here and elsewhere

Reply:
done